homeassistant/helpers/config_validation.py

```diff
@@ -642,6 +643,11 @@ def validator(value):
 PLATFORM_SCHEMA_BASE = PLATFORM_SCHEMA.extend({
 }, extra=vol.ALLOW_EXTRA)
 
+ENTITY_SERVICE_SCHEMA = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): comp_entity_ids,
+    vol.Optional(ATTR_AREA_ID): vol.All(ensure_list, [str]),
```

Comment:
why not `string`? Do you want allow None?

Reply:
it's an optional field. We don't allow `None` for `comp_entity_ids`